lib/jekyll/reader.rb

```diff
@@ -70,7 +70,7 @@ def retrieve_posts(dir)
     # dot_dirs - The Array of subdirectories in the dir.
     #
     # Returns nothing.
-    def retrieve_dirs(_base, dir, dot_dirs)
+    def retrieve_dirs(dir, dot_dirs)
```

Comment:
Sorry, no can do. These methods are not `private` which means this changes the method signature in a breaking way.

_If_ we want to do this, we need to wait for 4.0